.github/workflows/e2e-tests.yml

```diff
@@ -3,6 +3,8 @@ on:
   push:
     branches: [ main ]
   pull_request:
+  schedule:
```

Comment:
Wow, GitHub will just start running jobs like that? :o

Reply:
Only on default branch yeah https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#schedule

This is how we've been doing nightlies https://github.com/KittyCAD/modeling-app/blob/936549d60f2b98b1448c1523d59ff2cf397de5ca/.github/workflows/tag-nightly.yml#L7

Reply:
Amazing.